### PR TITLE
runbooks, Add CnaoNotReady alert

### PR DESCRIPTION
**Pull request by RamLavi**
This PR adds the CnaoNotReady alert runbook

### CODE REVIEW

File: docs/runbooks/CnaoNotReady.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# CnaoNotReady
+
+## Meaning
+
+Cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster.
```
Comment (qinqon) [inline docs/runbooks/CnaoNotReady.md:5]: I am not sure we need to add the CNAO is, looks like the alarm is happening when "additional networking components" are deployed.

They don't explain what kubevirt is here https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubeVirtComponentExceedsRequestedCPU.md#meaning

Maybe we can talk about CNAO CR

```
This alert fires when CNAO CR is not ready, which means that one of the deployed components is not ready.

File: docs/runbooks/CnaoNotReady.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+This alert fires when one or more of CNAO deployed components is not ready.
+
+## Impact
+
+Without CNAO components deployed and ready, the user might lack certain network components
+that are provided by this component.
```
Comment (qinqon) [inline docs/runbooks/CnaoNotReady.md:11]: Let's talk about the impact on use case for kubevirt.

```suggestion
Without CNAO components deployed and ready, not all the kubevirt networking scenarios can be executed
```

File: docs/runbooks/CnaoNotReady.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Diagnosis
+
+- Check for messages on the networkaddonsconfig CR status:
+	```
+	kubectl get networkaddonsconfig -o custom-columns="":.status.conditions[*].message
+	```
```
Comment (qinqon) [inline docs/runbooks/CnaoNotReady.md:18]: Maybe extracting the deployment/daemonSet have to be here and also readin the logs.

File: docs/runbooks/CnaoNotReady.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## Mitigation
+
+Identify which component is not ready, check the logs for the root cause and fix it.
```
Comment (qinqon) [inline docs/runbooks/CnaoNotReady.md:22]: Identify is the diagnosis, mitigation is something like "kill the component pods", or "kill CNAO pods", or re apply CR.

**Review by qinqon — CHANGES_REQUESTED** [review]

File: docs/runbooks/CnaoNotReady.md (hunk 1 above)
Comment (RamLavi) [reply docs/runbooks/CnaoNotReady.md:5]: > They don't explain what kubevirt is here https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubeVirtComponentExceedsRequestedCPU.md#meaning

but they do explain it on an HCO alert [KubevirtHyperconvergedClusterOperatorUSModification](https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubevirtHyperconvergedClusterOperatorUSModification.md) :) 

But I like your line about the CNAO CR, will change that, but let's continue to discuss if the CNAO intro thing is needed..

Comment (RamLavi) [reply docs/runbooks/CnaoNotReady.md:5]: As discussed on #18 , will keep this part.

File: docs/runbooks/CnaoNotReady.md (hunk 3 above)
Comment (RamLavi) [reply docs/runbooks/CnaoNotReady.md:17]: fixed.

**Comment by sradco** [comment]
@RamLavi Please run python automation/generate_pages_index.py
It re-generates the index file and add the new alert to the alerts list.

File: docs/runbooks/CnaoNotReady.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# CnaoNotReady
+
+## Meaning
+
+Cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster.
+This alert fires when CNAO CR is not ready, which means that one of the deployed components is not ready.
```
Comment (qinqon) [inline docs/runbooks/CnaoNotReady.md:6]: Let's add the name of the CR kind NetworkAddonsConfig

Comment (RamLavi) [reply docs/runbooks/CnaoNotReady.md:6]: https://github.com/kubevirt/monitoring/pull/21

**Pull request merged**